lib/gtfs_df/feed.rb

```diff
@@ -153,5 +110,64 @@ def filter(view)
       end
       self.class.new(filtered)
     end
+
+    private
+
+    def apply_filters(file, filters, original)
+      # Do not mutate the original
```

Comment:
How come we avoid mutating here but we do mutate in `prune!`? 

If useful, could we leave it as a comment in the code?